### ISSUE TITLE
Add Stork Search to benchmarks

### ISSUE THREAD

**Issue by Immortalin** [nextapps-de/flexsearch#311]
https://github.com/jameslittle230/stork

Will flexsearch prevail 👀?

**Comment by ts-thomas** [comment]
Yes, please provide a short example which fully runs in Browser (create index, add custom content to index, search query in index). Thanks a lot.
